lexicon.txt

```diff
@@ -85,7 +85,7 @@ busershutdown
 bwaitkeepalive
 bwinchange
 bwinscaling
-byt
+byte
```

Comment:
The lexicon contains only misspelled or unknown words.  `byte` is spelled correctly, so you should delete it from the lexicon.

```sh
$ echo byte | spell
$
```

Reply:
Done. Thanks.